Nimble/Expectation.swift

```diff
@@ -1,5 +1,25 @@
 import Foundation
 
+func expectTo<T, U where U: Matcher, U.ValueType == T>(expression: Expression<T>, matcher: U, to: String) -> (Bool, FailureMessage) {
```

Comment:
I'll create a style guide on the Quick Org if that sounds good to you. We can discuss and nitpick all the styles before applying them uniformly across Quick & Nimble.